assets/styles/login.css

```diff
@@ -194,6 +194,15 @@ label {
     font-size: .9em;
     padding: .55em 0
 }
+/* Specific styles for trust-device checkbox */
+label.two-factor-trust-device-label {
```

Comment:
Does this selector need to be so specific ? Do you think we can only keep `.two-factor-trust-device-label` ?